src/services/Form.ts

```diff
@@ -0,0 +1,35 @@
+import { make } from '../utils/make';
```

Comment:
why is this file stored in the `services` directory? I thought that "Service" — is a dude that can send form data somewhere. Like NotionService. Is it correct? What is service?